core/src/main/scala/cats/data/Ior.scala

```diff
@@ -142,6 +144,11 @@ sealed abstract class IorInstances extends IorInstances0 {
     def pure[B](b: B): A Ior B = Ior.right(b)
     def flatMap[B, C](fa: A Ior B)(f: B => A Ior C): A Ior C = fa.flatMap(f)
   }
+
+  implicit def bifunctor: Bifunctor[Ior] =
```

Comment:
Sorry, this is super nitpicky, but could you name this `iorBifunctor` just to match the convention of the instances above it?